WordPress/src/main/java/org/wordpress/android/ui/prefs/categories/detail/StateUtils.kt

```diff
@@ -0,0 +1,31 @@
+package org.wordpress.android.ui.prefs.categories.detail
```

Comment:
It is not very clear what state we're referring to in the `StateUtils` name. How about we rename it to `CategoryUiStateUtils`? In fact, I don't see `Utils` convention for naming classes holding `UiStates` (e.g. `ReaderCardUiState`), so `Utils` can be dropped from the name. Wdyt?

Reply:
@ashiagr Great points there. I agree. I will rename the `StateUtils` to `CategoryDetailUiState.kt` and the one in the listing screen to `CategoriesListingUIState.kt`. WDYT?

Reply:
Awesome! They sound so much better 🙌

Reply:
Excellent, Done in d23fcc0